Makefile

```diff
@@ -119,6 +116,17 @@ check_fmt:
 	    echo "Licenses are not formatted; run 'make fmt_license'"; exit 1 ;\
 	  fi \
 
+
+### fmt: Runs go fmt against code.  Borrowed from DWO
+fmt:
+  ifneq ($(shell command -v goimports 2> /dev/null),)
+	find . -not -path '*/\.*' -not -name '*zz_generated*.go' -name '*.go' -exec goimports -w {} \;
+  else
+	  @echo "WARN: goimports is not installed -- formatting using go fmt instead."
+	  @echo "      Please install goimports to ensure file imports are consistent."
+	  go fmt -x ./...
```

Comment:
Should we move the `go fmt -x ...` to outside of the if statement?

Reply:
wouldn't that be redundant if `goimports` has already been run?

Reply:
The [docs](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) say it can also be used to format